patches/tools-crates-gnrt-lib-gn.rs.patch

```diff
@@ -13,11 +13,41 @@ index 06ca0a9e6b6a144615bb158bda56ad8b2b516b6b..981b57207f2fbfabe1c56e8b81bc0ccb
              }
          }
          aliases.sort_unstable();
-@@ -537,6 +538,7 @@ fn target_os_to_condition(target_os: &str) -> &'static str {
+@@ -575,6 +576,7 @@ fn cfg_to_condition(cfg: &cargo_platform::Cfg) -> Condition {
+             "target_family" => target_family_to_condition(value),
+             "target_os" => target_os_to_condition(value),
+             "target_vendor" => target_vendor_to_condition(value),
++            "curve25519_dalek_backend" => Condition::AlwaysFalse,
```

Comment:
we undid this later, right? Would be helpful to squash these before review

Reply:
This got thrown out yes, but this happens later down the line.